pkg/dotenv/parse.go

```diff
@@ -42,8 +42,56 @@ var lineRe = regexp.MustCompile(
 // Ported from https://github.com/bkeepers/dotenv/blob/84f33f48107c492c3a99bd41c1059e7b4c1bb67a/lib/dotenv/parser.rb
 func Parse(data string) (map[string]string, error) {
 	var dotenv = make(map[string]string)
+	var lines = linesRe.Split(data, -1)
+	var inMultiline bool
+	var multilineValue string
+	var quoteChar byte
+
+	for i := 0; i < len(lines); i++ {
```

Comment:
I should be fine. But I would like to ask @zimbatm if we shouldn't rather use a dotenv parser library.